common/src/app/common/file_builder.cljc

```diff
@@ -222,9 +222,13 @@
 
 (defn close-artboard [file]
   (assert (nil? (:current-component-id file)))
-  (-> file
-      (assoc :current-frame-id root-frame)
-      (update :parent-stack pop)))
+
+  (let [parent-id (peek (get file :parent-stack))
```

Comment:
this probably can be expressed as `(-> file :parent-stack peek)`

Reply:
ok